arch/posix/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
+if (NOT CMAKE_HOST_UNIX OR CMAKE_HOST_APPLE)
+  message(FATAL_ERROR "The POSIX architecture only works with Linux hosts. If on Windows or macOS "
+          "consider a virtual machine")
```

Comment:
```suggestion
  message(FATAL_ERROR "The POSIX architecture only works with Linux hosts. If on Windows or macOS "
          "consider using a virtual machine to run a Linux guest")
```

Reply:
I changed it to:
```diff
+  message(FATAL_ERROR "The POSIX architecture only works on Linux. If on Windows or macOS "
+          "consider using a virtual machine to run a Linux guest.")
```
I hope you like it :)